pkg/server/config/config.go

```diff
@@ -59,6 +60,13 @@ type AuthenticatingProxy struct {
 	UsernameHeader string `yaml:"username_header"`
 }
 
+type Github struct {
+	Enabled        bool   `yaml:"enabled"`
+	AppId          int64  `yaml:"appId"`
+	InstallationId int64  `yaml:"installationId"`
+	PrivateKeyPath string `yaml:"privateKeyPath"`
```

Comment:
already commented on the README, but the yaml fields should be snake-cased (IMHO) to be consistent with the rest of the config options.